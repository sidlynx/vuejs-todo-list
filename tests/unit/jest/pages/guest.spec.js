import { shallowMount } from "@vue/test-utils";
import GuestComponent from "@/pages/guest";

describe('Component', () => {
    test('is a Vue instance', () => {
        const wrapper = shallowMount(GuestComponent)
        expect(wrapper.isVueInstance()).toBeTruthy()
        let a = true;
        expect(a).toBeTruthy();
        expect(true).toBe(true);
        console.log(wrapper.vm.mode)
        expect(wrapper).toMatchSnapshot()
    })
})
