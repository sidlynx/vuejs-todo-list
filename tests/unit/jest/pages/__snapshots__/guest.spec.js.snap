// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component is a Vue instance 1`] = `
<div class="guest">
  <div class="holder bordered">
    <div class="chooser">
      <div class="login fwb active">
        Login
      </div>
      <div class="register fwb">
        Register
      </div>
    </div>
    <!---->
    <div class="section login">
      <text-input-stub value="sidlynx@gmail.com" label="email" validationerrors=""></text-input-stub>
      <text-input-stub value="azerty" label="password" validationerrors="" type="password"></text-input-stub>
      <!---->
      <m-button-stub label="Login" state="info" style="float: right;"></m-button-stub>
    </div>
  </div>
</div>
`;
